spec/support/standard_setup.rb

```diff
@@ -76,186 +76,186 @@ def self.method_missing(method_name, *args)
+                                                       identifier: 'full_pprivate_foi_accepted'}.merge(a)) },
+      full_awdis_foi:               ->(a={}) { create(:case_with_response,
+                                                      :flagged_accepted,
+                                                      :press_office,
+                                                      {responder: responder_user,
+                                                       approver: disclosure_specialist_user,
+                                                       identifier: 'full_awdis_foi'}.merge(a)) },
+      full_responded_foi:           ->(a={}) { create(:responded_case,
+                                                      :flagged,
+                                                      :press_office,
+                                                      {responder: responder_user,
+                                                       identifier: 'full_responded_foi'}.merge(a)) },
+      full_closed_foi:              ->(a={}) { create(:closed_case,
+                                                      :flagged,
+                                                      :press_office,
+                                                      {identifier: 'full_closed_foi'}.merge(a)) },
 
-      std_unassigned_irc:           -> { create :compliance_review,
-                                                identifier: 'std_unassigned_irc' },
-      std_closed_irc:               -> { create :closed_compliance_review,
-                                                identifier: 'std_closed_irc' },
+      std_unassigned_irc:           ->(a={}) { create(:compliance_review,
+                                                      {identifier: 'std_unassigned_irc'}.merge(a)) },
+      std_closed_irc:               ->(a={}) { create(:closed_compliance_review,
+                                                      {identifier: 'std_closed_irc'}.merge(a)) },
 
-      std_unassigned_irt:           -> { create :timeliness_review,
-                                                identifier: 'std_unassigned_irt' },
-      std_draft_irt:                -> { create :accepted_timeliness_review,
-                                                identifier: 'std_draft_irt' },
-      std_closed_irt:               -> { create :closed_timeliness_review,
-                                                identifier: 'std_closed_irt' }
+      std_unassigned_irt:           ->(a={}) { create(:timeliness_review,
+                                                      {identifier: 'std_unassigned_irt'}.merge(a)) },
+      std_draft_irt:                ->(a={}) { create(:accepted_timeliness_review,
+                                                      {identifier: 'std_draft_irt'}.merge(a)) },
+      std_closed_irt:               ->(a={}) { create(:closed_timeliness_review,
+                                                      {identifier: 'std_closed_irt'}.merge(a)) }
```

Comment:
What's this a={}?

Reply:
This is an argument for the stabby lambda ... a is for attributes, which defaults to an empty hash.

Too obscure? The alternative is each one of these declarations turns into an even longer one, which I don't mind if this isn't obvious enough.

Reply:
I think it's fairly mystifying! you could do a comment at the top if you'd rather?

Reply:
Sure.

Reply:
Would turning 'a' into 'attributes' be sufficient?

Reply:
yep!

Reply:
So here's the alternative:

    @@cases = {
      std_unassigned_foi: ->(attributes={}) {
        create(:case, {identifier: 'std_unassigned_foi'}.merge(attributes))
      },
      std_awresp_foi: ->(attributes={}) {
        create(:assigned_case,
               {responding_team: responding_team,
                identifier: 'std_awresp_foi'}.merge(attributes))
      },
      std_draft_foi: ->(attributes={}) {
        create(:accepted_case,
               {responder: responder_user,
                identifier: 'std_draft_foi'}.merge(attributes))
      },
      std_draft_foi_late: ->(attributes={}) {
        create(:accepted_case,
               {responder: responder_user,
                received_date: 25.business_days.ago,
                identifier: 'std_draft_foi_late'}.merge(attributes))
      },

Reply:
I know it's long but it's a fair bit clearer!